portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,18 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+h2 {
+  text-align: center;
+}
+
+h3 {
+    font-style: italic;
+}
```

Comment:
Small thing: you want to keep a new line at the end of code. Some tools may complain if there isn't one.